src/jepsen/control/local.clj

```diff
@@ -0,0 +1,57 @@
+(ns jepsen.control.local
```

Comment:
Yeaaah, seeing things broken out like this really helps. I get the sense that there's a sort of polymorphic boundary here, where stuff like cd, sudo, and escaping would be shared between local & ssh implementations, and you could plug in local or ssh backends to the control system. I need to redesign control to get rid of the ridiculous dynamic state ball, but I'll keep this boundary in mind when I do that.